spec/annotations/semantics.yaml

```diff
@@ -5,6 +5,38 @@ x-extension:
   targetLink: "../spec-v1/csn-interop-effective"
 
 definitions:
+  "@Semantics.valueRange":
+    type: object
+    description: |-
+      Informs consumers about the value range of a CDS element, allowing to specify minimal and/or maximal values, and indicate whether these are exclusive or inclusive. 
+      This value range can be used for client-side validation of user input.
+    properties:
+      minimum:
+        type: string
```

Comment:
This probably has to be string, although its unintuitive. Reason is that JSON number would have precision loss and could for example not express very high/low Integer64 ranges.